plugins/out_s3/s3.c

```diff
@@ -252,6 +253,58 @@ static flb_sds_t concat_path(char *p1, char *p2)
     return dir;
 }
 
+static int read_seq_index(char *seq_index_file, uint64_t *seq_index,
```

Comment:
can you add a comment explaining how this function is used- why does it get two ints from the file?